Tools/i18n/makelocalealias.py

```diff
@@ -135,8 +135,8 @@ def check(data):
     args = parser.parse_args()
 
     data = locale.locale_alias.copy()
-    data.update(parse_glibc_supported(args.glibc_supported))
     data.update(parse(args.locale_alias))
+    data.update(parse_glibc_supported(args.glibc_supported))
```

Comment:
Because some of the X11 locale aliases conflict with the glibc ones. glibc obviously uses the glibc ones. This fixes problems where we would return encodings (like `en_IN.ISO8859-1`) that don't correspond to the C-level one.